mistralrs-core/src/pipeline/gguf.rs

```diff
@@ -356,7 +360,21 @@ impl Loader for GGUFLoader {
             info!("Debug is enabled, wrote the names and information about each tensor to `mistralrs_gguf_tensors.txt`.");
         }
 
-        let tokenizer = convert_ggml_to_hf_tokenizer(&model)?;
+        let ConversionResult {
+            tokenizer,
+            bos,
+            eos,
+            unk,
+        } = if paths.get_tokenizer_filename().to_string_lossy().is_empty() {
+            convert_ggml_to_hf_tokenizer(&model)?
+        } else {
+            ConversionResult {
+                tokenizer: get_tokenizer(paths.get_tokenizer_filename())?,
+                bos: None,
+                eos: None,
+                unk: None,
+            }
+        };
```

Comment:
I really don't want to deal with resolving conflicts again, this will be annoying for me to rebase against like the debug addition was.

Reply:
and... you just merged it 😑

Reply:
Ah, sorry! I didn't see the comment in time. Is there anything I can do to help it?

Reply:
Not really, but due to the number of commits and my change being early at the point straight after this addition, I get conflicts rippled through the history.

I'll probably be lazy and just use a merge commit to resolve instead of a proper rebase. I raised the PR before going to bed since there is a lot of activity recently on the files I was refactoring and it's starting to get tedious, so best to get what I've done so far in when you have time to review 👍

Reply:
Ok, sounds good, a merge commit is fine. I'll make a review now and then maybe you can do the rebase/merge?